### PR TITLE
Hard computer difficulty logic complete

### LINKED ISSUE #8: Make the hard computer shooting.
Right now it's random but it should be doing calculations.

### PR DESCRIPTION
**Pull request by evan-milton**
Removed some print statements and shortened 3 lines to 1 in Sink().

Fixes #8 and helps with #7.

### CODE REVIEW

**Comment by amedeekirk** [comment]
Looks good, Additional comments describing logic would be good for future.

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 1]
```diff
@@ -807,9 +821,9 @@ public static BattleshipModelUpdated Sink( Point shipStart, Point shipEnd, boole
             int y = shipStart.getDown();
 
             for ( int x = shipStart.getAcross(); x <= shipEnd.getAcross(); x++ ){
-                Point Shot = new Point();
-                Shot.setAcross(x);
-                Shot.setDown(y);
+                Point Shot = new Point(x, y);
+                //Shot.setAcross(x);
+                //Shot.setDown(y);
```
Comment (TrevorSpear) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:826]: Should have deleted the commented out lines

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 2]
```diff
@@ -362,26 +356,48 @@ private static BattleshipModelUpdated computerFireHard(BattleshipModelUpdated mo
...
+                        }else if( !alreadyShot( Up, model,false ) && Up.getDown() < 11){
+                            FireSpotComputer = new Point(Up.getAcross(), Up.getDown());
+                        }
+                    }
+                }
+            }
+        }
+
         j = model.getPlayerHits().size();
-        //Point Clipper = new Point((PArray[3].getStart().getAcross() + PArray[3].getEnd().getAcross())/2,(PArray[3].getStart().getDown() + PArray[3].getEnd().getDown())/2);
```
Comment (TrevorSpear) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:366]: Nice job

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 3]
```diff
@@ -362,26 +356,48 @@ private static BattleshipModelUpdated computerFireHard(BattleshipModelUpdated mo
...
-                //System.out.println(model.getShipByID(PArray[i].getName()).getHealth());
                 model.getShipByID(PArray[i].getName()).setHealth(PArray[i].getHealth() - 1);
-                System.out.println(model.getShipByID(PArray[i].getName()).getHealth());
 
                 //This is for sinking the ship.
                 if(model.getShipByID(PArray[i].getName()).getHealth() == 0){
                     model = Sink(model.getShipByID(PArray[i].getName()).getStart(), model.getShipByID(PArray[i].getName()).getEnd(), false, model);
-                    System.out.println("Sink: " + model.getShipByID(PArray[i].getName()).getHealth());
-                }else{
-                    System.out.println("Didn't Sink health: " + model.getShipByID(PArray[i].getName()).getHealth());
```
Comment (TrevorSpear) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:384]: Nice job getting rid of the trace statements

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 4]
```diff
@@ -362,26 +356,48 @@ private static BattleshipModelUpdated computerFireHard(BattleshipModelUpdated mo
...
+                            FireSpotComputer = new Point(Down.getAcross(), Down.getDown());
+
+                        }else if( !alreadyShot( Up, model,false ) && Up.getDown() < 11){
+                            FireSpotComputer = new Point(Up.getAcross(), Up.getDown());
+                        }
+                    }
+                }
+            }
+        }
+
```
Comment (TrevorSpear) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:387]: Can't wait to try it. Nice job naming of the variables it tells me where it is going to look at around the hit point

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 5]
```diff
@@ -319,17 +319,11 @@ private static String fireAtNew(Request req) {
 
 
                 model.addPointtoArray(FireSpot, model.getComputerHits());
-                //System.out.println("Player");
-                //System.out.println(model.getShipByID(CArray[i].getName()).getHealth());
```
Comment (TrevorSpear) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:323]: Thanks for getting rid of this commented out code

**Pull request merged**